test/behavior/tendrl/login.feature

```diff
@@ -0,0 +1,45 @@
+Feature: LOGIN
+
+  Scenario: Type username then delete
+    Given I open the Tendrl url
+    When  I type something in the username field
+    And   I delete what is in the username field
+    Then  The text "username is required" should appear under the username box
+
+  Scenario: Type password then delete
+    Given I open the Tendrl url
+    When  I type something in the password field
+    And   I delete what is in the password field
+    Then  The text "password is required" should appear under the password box
```

Comment:
Then  The text "password is required" should appear under the "password" box
this will make it generic